skopt/utils.py

```diff
@@ -292,3 +295,50 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+def transform_dims(dimensions, estimator="GP"):
```

Comment:
can we name it `transform_dimensions` or maybe even `normalize_dimensions`? Right now the name doesn't really tell you what the function does.